src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -14031,10 +13983,7 @@ void VRFCondenserEquipment::VRFOU_CalcCompC(EnergyPlusData &state,
                 Real64 CondHeat = Q_evap_req * C_cap_operation0 / this->RatedEvapCapacity; // 150130 To be confirmed
                 int CAPFT = this->OUCoolingCAPFT(CounterCompSpdTemp);
 
-                // Update Te' (SmallLoadTe) to meet the required evaporator capacity
-                MinOutdoorUnitTe = 6;
                 P_discharge = this->refrig->getSatPressure(state, T_discharge, RoutineName);
-
                 MinRefriPe = this->refrig->getSatPressure(state, -15, RoutineName);
                 MinOutdoorUnitPe = max(P_discharge - this->CompMaxDeltaP, MinRefriPe);
                 MinOutdoorUnitTe = this->refrig->getSatTemperature(state, max(min(MinOutdoorUnitPe, RefPHigh), RefPLow), RoutineName);
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:14040]:(style),[redundantAssignment],Variable 'MinOutdoorUnitTe' is reassigned a value before the old one has been used.